docker-compose-dev.yml

```diff
@@ -70,6 +70,46 @@ services:
     ports:
       - "8086:8086"
 
+  keycloak:
```

Comment:
Why `keycloak` here? This PR should be about `grafana` only. #283 should be about `keycloak`.

Reply:
I mean, once #283 is merged then this one ideally would be rebased to `development`, or are you just doing two in one shot so you don't have to rebase?

Reply:
Because this is a branch based off of https://github.com/xainag/xain-fl/pull/283 :D